Vuforia Spatial Toolbox.xcodeproj/project.pbxproj

```diff
@@ -708,7 +711,7 @@
 					"$(PROJECT_DIR)/Reality\\ Server/RE-server/node_modules/sharp/vendor/lib",
 					"$(PROJECT_DIR)/Reality\\ Server/RE-server/node_modules/sharp/vendor/lib",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "com.ptc.vuforia-spatial-toolbox";
+				PRODUCT_BUNDLE_IDENTIFIER = com.solaria.commDemo;
```

Comment:
I think this should stay com.ptc.vuforia-spatial-toolbox

Reply:
Unfortunately the base identifier doesn't have the notifications permission yet. If we don't change the identifier then this branch won't work out of the box.

It might be good to rename the orientation-options branch to commDemo since in some ways it's turning into the staging point for all of this

Reply:
Oh, ok! Makes sense! And yep, that's a good idea to rename the branch. 👍